modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -3,46 +3,49 @@
+Cluster balancing is crucial for optimal performance. Unbalanced clusters can saturate resources on one or more brokers, impacting  throughput and latency. Furthermore, a cluster with replicas on a down broker risks availability loss if more brokers fail, and a cluster that keeps losing brokers without healing eventually risks data loss. Redpanda provides various topic-aware tools to balance clusters for best performance. 
 
-With an Enterprise license, you can additionally enable _Continuous Data Balancing_ to continuously monitor broker and rack availability and disk usage. This enables self-healing clusters that dynamically balance partitions. It also continuously maintains adherence to rack-aware replica placement policy and self-heals after rack (or availability zone) failure or replacement. See xref:manage:cluster-maintenance/continuous-data-balancing.adoc[Configure Continuous Data Balancing].
-
-Cluster balancing protects you from unbalanced systems that saturate resources on one or more brokers. This can affect throughput and latency. Furthermore, a cluster with replicas on a down broker risks availability loss if more brokers fail, and a cluster that keeps losing brokers without healing eventually risks data loss.
-
-The following table summarizes the various balancing options in Redpanda:
+[.no-clip]
+|===
+| Topic-aware data balancer |Description
 
+| <<Partition leadership balancing>>
+| This balancer transfers the leadership of a broker's partitions to other replicas to avoid topic leadership hotspots on one or a few specific brokers in your cluster. 
+ +
+ +
+The partition leader regularly sends heartbeats to its followers. If a follower does not receive a heartbeat within a timeout, it triggers a new leader election. Redpanda also provides leadership balancing when brokers are added or decommissioned.
 
-.Cluster balancing options
-[cols="4,2,11"]
-|===
-|Balancer |xref:get-started:licenses.adoc[License] |Description
+| <<Partition replica balancing>>
+| This balancer moves partition replicas to avoid topic replica hotspots on one or a few specific brokers in your cluster. 
+ +
+ +
+Redpanda prioritizes balancing a topic's partition replica count on each broker while it's balancing the cluster's overall partition count. Because different topics in a cluster can have vastly different load profiles, this better distributes the workload evenly across brokers.
+ +
+ +
 
-|Partition leadership balancing
-|Community Edition, Enterprise Edition
-|Topic-aware balancer designed to distribute partition leadership across available nodes. This helps avoid topic leadership hotspots on one or a few specific nodes in your cluster.
+Partition balancing is invoked every 30 seconds by default. Redpanda also provides partition replica balancing when brokers are added or decommissioned.
```

Comment:
"Partition balancing is invoked every 30 seconds by default" - I would suggest leaving it out because it creates an impression that replica count balancing is invoked continuously (which is not true).

Reply:
Thanks, @ztlpn! I rephrased as: "Partition balancing is invoked automatically at regular intervals. Redpanda also provides partition replica balancing when brokers are added or decommissioned." @mattschumpert fyi